zerver/webhooks/sentry/view.py

```diff
@@ -220,12 +222,35 @@ def handle_deprecated_payload(payload: Dict[str, Any]) -> Tuple[str, str]:
     return (subject, body)
 
 
+def wrangle_webhook_payload(payload: Dict[str, Any]) -> Optional[Dict[str, Any]]:
```

Comment:
`wrangle` is a bit esoteric.  Maybe `transform`?

Reply:
Yeah, I'll change that. :+1: